### PR TITLE
[ZT] - add optional linux os version fields - os-version.md

### PR DESCRIPTION
**Pull request by imdawon**
* Added new linux version fields, how they are derived, and how to use them.

### CODE REVIEW

**Comment by ranbel** [comment]
When will the Linux option become available in the ZT dashboard?

**Review by kkrum — APPROVED** [review]

File: content/cloudflare-one/identity/devices/warp-client-checks/os-version.md [hunk 1]
```diff
@@ -71,3 +72,28 @@ Linux currently relies on the system Kernel version instead of a specific distro
...
+The following parameters are obtained via the `/etc/os-release` file. **Name** comes from the **ID** field, and **Revision** comes from the **VERSION_ID** field. 
+
+1. Open a Terminal window.
+1. Run the `cat /etc/os-release | grep "ID"` command to get OS identification fields that contain `ID`.
+
+    ```bash
+    cat /etc/os-release | grep "ID"
+    ```
+
+1. If the output of the above command contained the fields `ID=ubuntu` and `VERSION_ID=22.04`, our **Name** value would be `ubuntu` and **Revision** would be `22.04`.
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/warp-client-checks/os-version.md:92]: @dhens Do you have to format distro revision in SemVer. i.e. `22.04.0`?

Comment (kkrum) [reply content/cloudflare-one/identity/devices/warp-client-checks/os-version.md:92]: Sadly no, distro revision is a string and not consistent at all across the major Linux distros

**Comment by ranbel** [comment]
merging in #6191

**Pull request closed without merging**